internal/upgradecheck/header_test.go

```diff
@@ -500,12 +514,35 @@ func TestGetManagedClusters(t *testing.T) {
 		}
 		ctx, calls := setupLogCapture(ctx)
 		count := getManagedClusters(ctx, fakeClientWithOptionalError)
-		assert.Equal(t, len(*calls), 1)
+		assert.Assert(t, len(*calls) > 0)
 		assert.Assert(t, cmp.Contains((*calls)[0], `upgrade check issue: could not count postgres clusters`))
 		assert.Assert(t, count == 0)
 	})
 }
 
+func TestGetBridgeClusters(t *testing.T) {
+	ctx := context.Background()
+
+	t.Run("success", func(t *testing.T) {
+		fakeClient := setupFakeClientWithPGOScheme(t, true)
+		ctx, calls := setupLogCapture(ctx)
```

Comment:
Would you mind adding go-doc-comment descriptions for these and other helper functions in `helpers_test.go`?